unittests/resources/badchecks/notacheck.py

```diff
@@ -1 +1 @@
-import _foo
```

Comment:
This is needed to check a test having a Python file as resources and this files does an import that does not exist. We had a bug before with a test doing exactly that and crashing the framework.